src/ui/component/card/card.scss

```diff
@@ -24,6 +24,10 @@ $backgrounds: (
           background-color: themed(card-background-hover);
         }
 
+        &:focus {
+          background-color: themed('card-background-pressed');
+        }
```

Comment:
I am probably missing something, because during testing I was not able to focus on the card, but only the child button. We might want to use `:focus-within` here instead of `:focus`. It tried with `focus-within`, It worked for me but only outside of the background loop.

Reply:
Sorry for misleading you, your implementation is correct because we must only target interact cards. Since these cards are actually `div` element, they are not focusable by a user. We might create an issue in order to make them accessible by either transforming these divs in `button` tag or providing them a `tabindex`.